Dockerfile

```diff
@@ -462,6 +462,38 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-license-scan as sdk-cargo-deny
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/cargo-deny && \
+  chown -R builder:builder /usr/share/licenses/cargo-deny
+
+ARG DENYVER="0.6.2"
```

Comment:
I read the changelogs beyond 0.6.2 and I think it'd be good to update, though we don't have to do it immediately.  We'd probably want to pass `--all-features` to make sure we're not missing anything, since that's no longer the default, and probably make sure `bans.wildcards` is enabled.  SPDX data has been updated in later versions, so it'd be good to get that.

Reply:
Opened https://github.com/bottlerocket-os/bottlerocket-sdk/issues/50 to track this.